_include/q/quickstart-scala-SNAPSHOT.sh

```diff
@@ -21,10 +21,20 @@
 
 PACKAGE=quickstart
 
+function extract_parameter() {
+  if [[ "$#" != 1 ]]; then
+    echo "Fatal error: No parameter or too many parameter passed: $@"
+    exit 1;
+  fi
+  awk -F'"' '/'$1' = "/{print $2}' docs/config.toml
+}
+
+defaultFlinkVersionSnapShot="$(extract_parameter 'FlinkStableShortVersion')-SNAPSHOT"
+
 mvn org.apache.maven.plugins:maven-archetype-plugin:2.4:generate	\
   -DarchetypeGroupId=org.apache.flink 								\
   -DarchetypeArtifactId=flink-quickstart-scala						\
-  -DarchetypeVersion=${1:-1.17-SNAPSHOT}									\
+  -DarchetypeVersion=${defaultFlinkVersionSnapShot}									\
```

Comment:
```suggestion
  -DarchetypeVersion=${1:-${defaultFlinkVersionSnapShot}}									\
```
you omitted the parameter here